packages/cache/src/cache.ts

```diff
@@ -61,13 +61,15 @@ export function isFeatureAvailable(): boolean {
  * @param paths a list of file paths to restore from the cache
  * @param primaryKey an explicit key for restoring the cache
  * @param restoreKeys an optional ordered list of keys to use for restoring the cache if no cache hit occurred for key
+ * @param crossOsEnabled an optional boolean enabled to restore on windows any cache created on any platform
  * @param downloadOptions cache download options
  * @returns string returns the key for the cache hit, otherwise returns undefined
  */
 export async function restoreCache(
   paths: string[],
   primaryKey: string,
   restoreKeys?: string[],
+  crossOsEnabled = false,
```

Comment:
Thinking about the name - is `crossOsCompatibility` better name? "Enabled" does convey that this is kind of a feature flag (which it is!) but from user perspective compatibility is what is important and how it is rolled out.
Thoughts?

Reply:
`crossOsEnabled` feels better, as compatibility is something largely user has to think about when enabling this as to if the cache contents are compatible or not

Reply:
My two cents here, I agree with not using compatibility since it elludes to making the contents compatible. While `crossOsEnabled` does work, it feels unconventional since we prefer using present/present-perfect tense. How about `enableCrossOs`?

Reply:
I see your points. But I also feel that `enableCrossOs` is not giving complete clarity on what it is doing. That's why I was adding `Compatibility`. How about we call it what is precisely is - `enableCrossOsArchive`?

Reply:
`enableCrossOsArchive` is much more precise yes 👍

Reply:
Done